src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -109,6 +119,13 @@ class BaseCreateCourseForm extends React.Component {
             label={<FieldLabel text="Organization" required />}
             required
           />
+          <Field
+            name="source"
+            component={RenderSelectField}
+            options={this.processSources(sources)}
+            label={<FieldLabel text="Source" required />}
+            required
+          />
```

Comment:
Isn't this still showing the dropdown? What about the plan of hiding this and sending a default value?

Reply:
it's in a followup PR
### Followup PR link
https://github.com/openedx/frontend-app-publisher/pull/845